var/spack/repos/builtin/packages/py-easybuild-framework/package.py

```diff
@@ -0,0 +1,40 @@
+##############################################################################
+# Copyright (c) 2017, Kenneth Hoste
+#
+# This file is part of Spack.
+# Created by Kenneth Hoste, kenneth.hoste@gmail.com
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyEasybuildFramework(PythonPackage):
+    """
+    The core of EasyBuild, a software build and installation framework
+    for (scientific) software on HPC systems.
+    """
+
+    homepage = 'http://hpcugent.github.io/easybuild/'
+    url      = 'https://pypi.python.org/packages/fa/00/6a47862b38e6d921071d305deab4c494ae2eae58556c90c95e520fea28b9/easybuild-framework-3.1.2.tar.gz'
+
+    version('3.1.2', '283bc5f6bdcb90016b32986d52fd04a8')
+
+    depends_on('python', type='run')
+    depends_on('py-setuptools', type=('build', 'run'))
+    depends_on('py-vsc-base', type='run')
+    depends_on('py-vsc-install', type='run')  # only requires for tests (python -O -m test.framework.suite)
```

Comment:
well, it's not really for testing only, you can also run the EasyBuild framework test suite (partially) after installation, so it really *is* a runtime dependency

if you'd try running `python -O -m test.framework.suite`, it would go down hard if `vsc-install` is not available.